dashboard/app/controllers/scripts_controller.rb

```diff
@@ -100,19 +100,12 @@ def destroy
   end
 
   def edit
-    beta = params[:beta].present?
-    if @script.script_levels.any?(&:has_experiment?)
-      beta = false
-      beta_warning = "The beta Script Editor is not available, because it does not support level variants with experiments."
-    end
```

Comment:
my first thought is that the has_experiment check seems ok to remove, because no migrated scripts will have experiments to begin with. then, we plan to implement support for this before we hard-launch in June, as tracked in roadmap/jira. 

However, my next thought is that it might be better to find a way to protect ourselves in case we forget and migrate a script that has experiments before implementing support for experiments. what are your thoughts on doing something like this here?
```ruby
raise "the new script editor does not support level variants with experiments" if @script.is_migrated && @script.script_levels.any?(&:has_experiment?) 
```
this seems like the safest bet to me, because it prevents the editor page from causing data loss, but I'm open to skipping it if there's some other way we think this will be taken care of elsewhere somehow.